vocabularies/geological-sites.ttl

```diff
@@ -194,12 +172,12 @@ gsqst:wellbore-interval a skos:Concept ;
     skos:prefLabel "Wellbore Interval"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/geological-sites> .
 
-gsqst:project-site a skos:Concept ;
+gsqst:survey-site a skos:Concept ;
     skos:altLabel "Project"@en ;
     skos:definition "The location of a survey."@en ;
     skos:inScheme <http://linked.data.gov.au/def/geological-sites> ;
-    skos:notation "pj" ;
-    skos:prefLabel "Project Site"@en ;
+    skos:notation "ss" ;
+    skos:prefLabel "Survey Site"@en ;
```

Comment:
"ss" is the prefix for survey type: seismic survey
Prefixes (notations) should be unique throughout the ecosystem of features, site, survey, sample, obs, results etc.
Helps to ensuring PID distinctiveness. Since it is for a machine read purpose (rather than human interrogation) there is no harm leaving the notation as pj for uniqueness and leaving project site as an alternative label. Otherwise this makes sense to amalgamate project/survey sites to incorporate seismic surveys and let the survey type deal with the survey type.

Reply:
Good pick up @KellyVance, though we should be k because there are no more seismic surveys, just geophysical surveys. We need to get you up to speed on the survey front since we finally got to the bottom of surveys in sosa and have a new way to store "multi-type" surveys.

Reply:
Thats fine. we will have to edit the survey type. but in the meantime still a good idea to keep notations unique at any point in time. We should push this along and get the approvals to cclose out this pull request